src/main/resources/lang/locale-en.properties

```diff
@@ -1,5 +1,5 @@
 language=English
-disambiguation=disambiguation,hndis
+disambiguation=disambiguation,hndis,Airport disambiguation,Biology disambiguation,Call sign disambiguation,Caselaw disambiguation,Chinese title disambiguation,Genus disambiguation,Geodis,Hndis,Hndis-cleanup,Hospital disambiguation,Letter disambiguation,Letter-NumberCombDisambig,Mathematical disambiguation,Mil-unit-dis,Numberdis,Phonetics disambiguation,Road disambiguation,School disambiguation,Species Latin name disambiguation,Disambig,dab,Disamb
```

Comment:
There's no script, I had to manually collect those from https://en.wikipedia.org/wiki/Template:Disambiguation, there seems to be no API for that.